src/psyclone/psyGen.py

```diff
@@ -2780,6 +2808,126 @@ def validate(self, node, options=None):
         :type options: Optional[Dict[str, Any]]
         '''
         # pylint: disable=unused-argument
+        # TODO 2668: options are now deprecated:
+        if options is not None:
+            print("Deprecation Warning: The options parameter to "
```

Comment:
"PSyclone deprecation warning" again please. You could store this message text as a class variable perhaps.

Reply:
I added it to the class now - not sure if its a class or instance variable though but it seems to work.